### ISSUE TITLE
Why does not the model converge when use the softmax instead of sigmoid  in the output

### ISSUE THREAD

**Issue by yianzhongguo** [ellisdg/3DUnetCNN#155]
@ellisdg Hi, sir. I found the sigmoid function was used in the output of model. When I use the softmax function to replace it, the training prcoess ended with the resut: the loss was about -0.04. It puzzled me.
Why is it?Thank you!
